fhir-server/src/main/java/com/ibm/fhir/server/filter/rest/FHIRRestServletFilter.java

```diff
@@ -261,6 +279,30 @@ private HTTPReturnPreference computeReturnPref(ServletRequest request, HTTPHandl
         return returnPref;
     }
 
+    /**
+     * Checks if the FHIR version parameter in the specified HTTP header is valid.
+     * @param headerName the name of the header
+     * @param requestHeaders the headers
+     * @return the error message if FHIR version parameter in not valid, otherwise null
+     */
+    private String checkFhirVersionParameter(String headerName, Map<String, List<String>> requestHeaders) throws FHIRException {
+        for (String headerValue : requestHeaders.getOrDefault(headerName, Collections.emptyList())) {
```

Comment:
I think we should ignore multiple headers, and only check  the first, reject multiples.

Reply:
Updated to reject multiple different versions for Content-Type, but still checks fhirVersion on each header.